examples/crm-full-stack/nodejs-backend/README.md

```diff
@@ -0,0 +1 @@
+# node-backend-typescript-prisma
```

Comment:
Did you mean to have more here? Or should this README not be here at all?

Reply:
Good catch - removed this README, kept one for the parent directory